README.md

```diff
@@ -1,34 +1,42 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Hololive EN Fan website
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+This repository holds the website for [holoen.fans](https://holoen.fans), which was started by the [Hololive EN Fan Discord](https://discord.gg/holoenfans).
 
-## Getting Started
+## Stack
 
-First, run the development server:
+* [Next.JS](https://nextjs.org) [React](https://reactjs.org/)
```

Comment:
I don't think adding React here is not necessary since NextJS is a React framework but that doesn't mean that React is the same as Next.

```suggestion
* [Next.JS](https://nextjs.org)
```

Reply:
True